test/test_linalg.py

```diff
@@ -575,6 +575,63 @@ def run_test(upper):
         for upper in [True, False]:
             run_test(upper)
 
+    @skipCUDAIfNoMagmaAndNoCusolver
+    @skipCPUIfNoLapack
+    @dtypes(torch.float32, torch.float64, torch.complex64, torch.complex128)
+    def test_cholesky_with_info(self, device, dtype):
```

Comment:
test_cholesky_ex? That will make it easier for people to find the test(s) associated with the operator